components/script/dom/document.rs

```diff
@@ -2739,26 +2748,26 @@ impl Document {
             self.has_pending_animated_image_update.set(false);
         }
 
+        self.current_layout_epoch
```

Comment:
nit(non-blocking): I am not sure about the naming here, as we still bump the epoch even if we skip layout (well at least we will be when https://github.com/servo/servo/pull/38991 lands). Maybe we should name it "rendering epoch" (because of update the rendering)?

Reply:
That makes sense. I've done this.